rules/windows/command_and_control_dns_tunneling_nslookup.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/11"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of Nslookup process execution with explicit query type from the same host. This may indicate a
+Command and Control activity via DNS Protocol.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential DNS Tunneling with Nslookup"
+references = ["https://unit42.paloaltonetworks.com/dns-tunneling-in-the-wild-overview-of-oilrigs-dns-tunneling/"]
+risk_score = 47
+rule_id = "3a59fc81-99d3-47ea-8cd6-d48d561fca20"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "threshold"
+
+query = '''
+event.category:process and event.type:(start or process_started) and process.name:nslookup.exe and process.args:(-querytype=* or -qt=* or -q=* or -type=*)
```

I don't think that `event.type:process_started` is needed anymore. We can just use `start`.
```suggestion
event.category:process and event.type:start and process.name:nslookup.exe and process.args:(-querytype=* or -qt=* or -q=* or -type=*)
```